packages/stable/src/types.ts

```diff
@@ -1,4 +1,30 @@
-// Copyright 2019 Cognite AS
+// Copyright 2020 Cognite AS
+
+import {
+  CogniteExternalId,
+  CogniteInternalId,
+  Cursor,
+  ExternalId,
+  FilterQuery,
+  IdEither,
+  InternalId,
+  Limit,
+} from '@cognite/sdk-core';
+
+export {
+  CogniteExternalId,
+  CogniteInternalId,
+  Cursor,
+  ExternalId,
+  FilterQuery,
+  IdEither,
+  InternalId,
+  Limit,
+  LogoutUrl,
+  CursorResponse,
+  ItemsWrapper,
+  ClientOptions,
+} from '@cognite/sdk-core';
```

Comment:
why there are two similar exports?

> refactor: export enough of cores types for stable and beta to compile

also, all current types from core should be exported (because it used to be like that)

(unless you have objections)

Reply:
I agree all current types should be exported, but it looks very bad to export * from core in types.ts. I also agree the overlap between imports and exports looks weird. We could perhaps in typescript 3.8 do export types, but even then, not all types from core are supposed to be exported from types.ts in stable (such as BaseCogniteClient, yes it's a type, but not in this sense). Do we have to manually export every type in `core/src/types.ts`?

Reply:
I meant export all types that are in `@cognite/sdk-core/src/types.ts` ofc
not everything))
types.ts is basically consisting of types that were previously exported anyway (it's around 20 of them)

Reply:
I did a new pull request for this, just to avoid messing with the base of 3 other PRs